core/imageroot/usr/local/sbin/apply-vpn-routes

```diff
@@ -85,19 +82,21 @@ for iface in ipaddr_reply:
 errors = 0
 valid_destinations = []
 for node_id in destmap:
+    peer_host, peer_port = resolve_endpoint(peers[node_id].get('endpoint', ''))
+    allowed_ips = destmap[node_id] - {peer_host}
```

Comment:
Bit unsure about this, do you wrap the `peer_host` in a `dict` to run a difference between `dict`s?